Gems/AtomLyIntegration/CommonFeatures/Assets/Scripts/Profiling/OutputProfileData.lua

```diff
@@ -0,0 +1,134 @@
+----------------------------------------------------------------------------------------------------
+--
+-- Copyright (c) Contributors to the Open 3D Engine Project.
+-- For complete copyright and license terms please see the LICENSE at the root of this distribution.
+--
+-- SPDX-License-Identifier: Apache-2.0 OR MIT
+--
+--
+--
+----------------------------------------------------------------------------------------------------
+
+local OutputProfileData = 
+{
+    Properties = 
+    {
+        FrameDelayCount = 100,
+        FrameCaptureCount = 100,
+        ProfileName = "LevelFrameTiming",
+        QuitOnComplete = true,
+    },
+    frameCount = 0,
+    frameDelayCount = 0,
+    frameCaptureCount = 0,
+    captureCount = 0,
+    quitOnComplete = false,
+    profileName = "UninitializedName",
+    outputFolder = "UninitializedPath",
+    cpuTimingsOutputPath = "UninitializedPath",
+    captureInProgress = false,
+    active = false,
+};
+
+local FrameTimeRecordingActiveRegistryKey <const> = "/O3DE/Performance/FrameTimeRecording/Activate"
+local FrameDelayCountRegistryKey <const> = "/O3DE/Performance/FrameTimeRecording/DelayCount"
+local FrameCaptureCountRegistryKey <const> = "/O3DE/Performance/FrameTimeRecording/CaptureCount"
+local ProfileNameRegistryKey <const> = "/O3DE/Performance/FrameTimeRecording/ProfileName"
+local QuitOnCompleteRegistryKey <const> = "/O3DE/Performance/FrameTimeRecording/QuitOnComplete"
+local SourceProjectUserPathRegistryKey <const> = "/O3DE/Runtime/FilePaths/SourceProjectUserPath"
+local ConsoleCommandQuitRegistryKey <const> = "/Amazon/AzCore/Runtime/ConsoleCommands/quit"
+
+function OutputProfileData:TryDisconnect()
+    if (self.active) then
+        self.active = false
+        self.profileCaptureNotificationHandler:Disconnect()
+        self.tickHandler:Disconnect()
+    end
+end
+
+function OutputProfileData:TryQuitOnComplete()
+    if (self.quitOnComplete) then
+        g_SettingsRegistry:SetString(ConsoleCommandQuitRegistryKey, "")
```

Comment:
Is this command supposed to quit the application? It seems to be setting a registry value.

Reply:
It does in fact quit the application. By sending a console command through the registry.